bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,28 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    json_embeds = []
```

Comment:
```suggestion
    embed_data = []
```
I didn't quite understand what `json_embeds` was meant to do without analysing the rest of the code. Something like `embed_data` would make this a lot more clear. (Remember to update all references to this variable name)

Reply:
Yeah that makes more sense.